docs/BUSINESS.md

```diff
@@ -0,0 +1,60 @@
+- For larger teams, it would be appropriate to have this profile managed by chat bot that can reply to some simple questions, and to add support agents, based on their availability and the questions asked. These scenarios would require programming a chat bot, and we are currently working simplify it.
+
+In any case, it is important that the client application remains running and connected to the Internet for you to receive support requests.
+
+## Customer broadcasts
+
+While currently supported only via CLI clients (or via chat console in desktop and mobile clients), it can be used to broadcast important announcements to all connected customers. We will be adding this feature to desktop clients soon. We use it to broadcast release updates to a very large number of users who are connected to our own support profile.
+
+## Community groups and promotion in group directory
+
+In addition to providing support to clients individually, you can create a community group, and promote it via our experimental and growing [directory of public groups](./DIRECTORY.md). Community groups require ongoing moderation.
+
+## Limitations
+
+With all advantages in privacy and security of e2e encryption in SimpleX Chat, there are some important limitations:
+- **protecting your data from loss is your responsibility**. This is the price of privacy - if you lose your device, or database passphrase, there is absolutely no way we would be able to support you to recover access. There are ways to work around these limitations.
+- **you cannot access the same profile from multiple devices**. For all communication products it's a basic expectation, and yet there is not a single one that delivered it without some very serious privacy and security compromises. Better solutions are possible, and we will be implementing it, but reasonably secure approach is much more complex to implement than what is affordable at the current stage. You can access mobile or CLI profile from desktop, and the latter allows to use one profile by multiple people in turns, as we explain below.
+- **your owner role in the groups cannot be restored if you lose the device**. The solution is to create owner profiles on multiple devices for all your important groups. This way if you lose device or data for one of profiles, you won't lose control of the group, and you can add a new one. Think about it as about keys to your cryptowallet.
+- **current groups are highly experimental**. Message delivery can be delayed of fail in some cases, lists of members can be out of sync. There are approaches to make them more stable we use for our groups.
```

Comment:
```suggestion
- **current groups are highly experimental**. Message delivery can be delayed or fail in some cases, lists of members can be out of sync. There are approaches to make them more stable we use for our groups.
```

Reply:
mention that it doesn't affect basic scenario 1-1 business chat and overall less likely?